pkg/algorithm/topology_aware_scheduler.go

```diff
@@ -308,24 +308,24 @@ func findNodesForPods(cv clusterView, gpuNums []int32) (pickedNodeIndices []int3
 // findGpusInNode finds a set of GPUs with the best affinity in a node for a pod.
 func findGpusInNode(
 	n Cell,
-	gpuNum int32,
+	skuNum int32,
 	p CellPriority,
 	availableGpus CellList,
-	levelGpuNum map[CellLevel]int32) (CellList, CellList) {
+	levelSkuNum map[CellLevel]int32) (CellList, CellList) {
 
 	// indices of the currently picked GPUs
-	currentGpuIndices := make([]int32, gpuNum)
+	currentGpuIndices := make([]int32, skuNum)
 	// affinity of the currently picked GPUs, defined as the lowest common ancestor
 	// of the GPUs in the cell hierarchy (lower level means better affinity)
-	currentAffinity := make(CellList, gpuNum)
+	currentAffinity := make(CellList, skuNum)
 	// GPUs with the best affinity ever seen
```

Comment:
Still some leftovers:
// GPUs with the best affinity ever seen
bestAffinityGpus 
bestAffinityGpuIndices

Reply:
updated